modules/programs/yazi.nix

```
@@ -77,16 +77,16 @@ in {
       default = { };
       example = literalExpression ''
         {
-          input.keymap = [
-            { exec = "close"; on = [ "<C-q>" ]; }
-            { exec = "close --submit"; on = [ "<Enter>" ]; }
-            { exec = "escape"; on = [ "<Esc>" ]; }
-            { exec = "backspace"; on = [ "<Backspace>" ]; }
+          input.keymap_prepend = [
```

I believe the correct attribute to use here is `prepend_keymap` (see [keymap.toml](https://yazi-rs.github.io/docs/configuration/keymap/))

```suggestion
          input.prepend_keymap = [
```

---

Oops. Fixed.